### PR TITLE
resolved old links

### PR DESCRIPTION
**Pull request by kjeom**
## Proposed changes

- Updated the old links (kip, truffle)

## Types of changes

Please put an x in the boxes related to your change.

- [x] Minor Issues and Typos
- [ ] Major Content Contribution
- [ ] Others

## Checklist

*Put an x in the boxes that apply. You can also fill these out after creating the PR. If you're unsure about any of them, don't hesitate to reach out. We're here to help! This is simply a reminder of what we are going to look for before merging your code.*

- [x] I have read the [CONTRIBUTING GUIDELINES](https://github.com/klaytn/klaytn-docs/blob/master/CONTRIBUTING.md)
- [x] I have signed the [CLA](https://cla-assistant.io/klaytn/klaytn-docs)
- [ ] I have added necessary documentation (if appropriate)
- [ ] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

File: docs/smart-contract/token-standard.md [hunk 1]
```diff
@@ -59,7 +59,7 @@ Based on the interface above, developers may customize tokens by adding new feat
 
 For more information, refer to the official [KIP-7 documentation](https://kips.klaytn.foundation/KIPs/kip-7).
 
-* An example implementation is available at [https://github.com/klaytn/klaytn-contracts/blob/master/contracts/token/KIP7/KIP7Token.sol](https://github.com/klaytn/klaytn-contracts/blob/master/contracts/token/KIP7/KIP7Token.sol).
+* An example implementation is available at [https://github.com/klaytn/klaytn-contracts/blob/master/contracts/KIP/token/KIP7/KIP7.sol](https://github.com/klaytn/klaytn-contracts/blob/master/contracts/KIP/token/KIP7/KIP7.sol).
```
Comment (toniya-klaytn) [inline docs/smart-contract/token-standard.md:62]:
```suggestion
* An example implementation is available at [https://github.com/klaytn/klaytn-dev-sandbox/blob/dev/contracts/token/KIP7/KIP7Token.sol](https://github.com/klaytn/klaytn-dev-sandbox/blob/dev/contracts/token/KIP7/KIP7Token.sol).
```

File: docs/smart-contract/token-standard.md [hunk 2]
```diff
@@ -125,7 +125,7 @@ Based on the interface above, developers may customize tokens by adding new feat
 
 For more information, refer to the official [KIP-17 documentation](https://kips.klaytn.foundation/KIPs/kip-17).
 
-* An example implementation is available at [https://github.com/klaytn/klaytn-contracts/blob/master/contracts/token/KIP17/KIP17Token.sol](https://github.com/klaytn/klaytn-contracts/blob/master/contracts/token/KIP17/KIP17Token.sol).
+* An example implementation is available at [https://github.com/klaytn/klaytn-contracts/blob/master/contracts/KIP/token/KIP17/KIP17.sol](https://github.com/klaytn/klaytn-contracts/blob/master/contracts/KIP/token/KIP17/KIP17.sol).
```
Comment (toniya-klaytn) [inline docs/smart-contract/token-standard.md:128]:
```suggestion
* An example implementation is available at [https://github.com/klaytn/klaytn-dev-sandbox/blob/dev/contracts/token/KIP17/KIP17Token.sol](https://github.com/klaytn/klaytn-dev-sandbox/blob/dev/contracts/token/KIP17/KIP17Token.sol).
```

**Review by toniya-klaytn — COMMENTED** [review]
Kindly consider the suggesstions

**Review by terrikwak — APPROVED** [review]

File: docs/smart-contract/token-standard.md (hunk 2 above)
Comment (kjeom) [reply docs/smart-contract/token-standard.md:128]: @toniya-klaytn Could you comment the difference between the two repo.?

Comment (toniya-klaytn) [reply docs/smart-contract/token-standard.md:128]: @kjeom  reverting my suggestion as I gave example link on how to use KIP17.

**Review by toniya-klaytn — APPROVED** [review]

**Comment by toniya-klaytn** [comment]
@kjeom reverting my suggestion as I gave example link on how to use KIP17. I have approved it.

**Pull request merged**